velox/connectors/hive/storage_adapters/test_common/InsertTest.h

```diff
@@ -29,7 +29,31 @@
 namespace facebook::velox::test {
 
 class InsertTest : public velox::test::VectorTestBase {
- public:
+ protected:
+  void SetUp(
```

Comment:
Move the `register `and `unregister ` code from `GcsInsertTest `and `S3InsertTest ` into `InsertTest ` to reduce the code duplicated.